kissmanga-downloader.py

```
@@ -66,6 +67,44 @@ def init_driver():
     return driver
 
 
+def gather_xml_info(driver, title_text):
+    """
+    Dig through the first page and gather data for a Comicinfo.xml file
+    """
+    xml_root = ET.Element('ComicInfo',
+                          {'xmlns:xsi':'http://www.w3.org/2001/XMLSchema-instance',
+                           'xmlns:xsd':'http://www.w3.org/2001/XMLSchema'})
+
+    xml_series = ET.SubElement(xml_root, 'Series')
+    xml_series.text = title_text
+    try:
+        author = driver.find_element_by_xpath("//div[contains(@class, 'author-content')]/a")
+        xml_author = ET.SubElement(xml_root, 'Writer')
+        xml_author.text = author.get_attribute('innerHTML')
+    except ET.NoSuchElementException:
+        pass
```

We should probably `print` something as a form of logging in the `except` clauses of these `try...except` blocks. Applies for all `except` clauses which have `pass`

---

For some of these, the idea is that IF the manga has "Writer" listed, we create a writer XML segment, but if not, we really do nothing, we don't want that XML tag at all in the resulting xml, so we just silently skip.. I guess we could say "no writer found", but, it ends up not being really an error, it's just they didn't fill that in on the site.